### PR TITLE
Amended pom.xml to build only compliance-action-completed model, Add version & Makefile files

### PR DESCRIPTION
**Pull request by Zain-Abbas1**
Add Makefile and version file to enable repo to be built on Concourse

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>uk.gov.companieshouse</groupId>
     <artifactId>kafka-models</artifactId>
-    <version>unversioned</version>
```
Comment (jsimons1989) [inline pom.xml]: Can you keep this as unversioned please

File: pom.xml [hunk 2]
```diff
@@ -69,6 +69,9 @@
                         <configuration>
                             <stringType>String</stringType>
                             <sourceDirectory>${project.basedir}/chs-kafka-schemas/schemas</sourceDirectory>
+							<includes>
+								<include>compliance-action-completed.avsc</include>
+							</includes>
```
Comment (jsimons1989) [inline pom.xml]: Can you fix the formatting please

File: pom.xml (hunk 1 above)
Comment (Zain-Abbas1) [reply pom.xml]: Done

File: pom.xml (hunk 2 above)
Comment (Zain-Abbas1) [reply pom.xml]: Done

**Pull request merged**